src/DotLiquid.Tests/ConditionTests.cs

```diff
@@ -64,7 +64,7 @@ public void TestContainsWorksOnStrings()
         public void TestContainsWorksOnArrays()
         {
             _context = new Context(CultureInfo.InvariantCulture);
-            _context["array"] = new[] { 1, 2, 3, 4, 5 };
+            _context["array"] = new long[] { 1, 2, 3, 4, 5 };

```

Comment:
It was wrong for me to change the test input. Instead the library needs to continue working with int and work with long seamlessly.